src/Microsoft.IdentityModel.Logging/IdentityModelTelemetryUtil.cs

```diff
@@ -135,5 +135,52 @@ internal static bool UpdateDefaultTelemetryData(string key, string value)
             telemetryData[key] = value;
             return true;
         }
+
+        //internal static void IncrementConfigurationManagerCounter(
```

Comment:
Why are these commented out?

Reply:
I moved them elsewhere and didn't clean up that file yet